### ISSUE TITLE
resolve warnings in `setup-meta-infrastructure` module

### ISSUE THREAD

**Issue by ketzacoatl** [fpco/terraform-aws-foundation#190]
```
Warning: output "admin-group-name": must use splat syntax to access aws_iam_group.admin attribute "name", because it has "count" set; use aws_iam_group.admin.*.name to obtain a list of the attributes across all instances



Warning: output "power-user-group-name": must use splat syntax to access aws_iam_group.power-user attribute "name", because it has "count" set; use aws_iam_group.power-user.*.name to obtain a list of the attributes across all instances



Warning: output "setup-mfa-group-name": must use splat syntax to access aws_iam_group.setup-mfa attribute "name", because it has "count" set; use aws_iam_group.setup-mfa.*.name to obtain a list of the attributes across all instances

```

**Comment by psibi** [comment]
This seems to be fixed by @Magicloud in 0.12 branch in this commit: 45788f0dc12a21248e0e7afa17b93d9e97288624

**Comment by ketzacoatl** [comment]
Support for v0.12 has been merged to master and released as `v0.9.0`
